src/configuration/structures.ts

```diff
@@ -51,6 +51,7 @@ export interface BotFields {
   max_chats_count?: number;
   groups?: GroupConfig[];
   work_scheduler?: WorkScheduler;
+  owner_client_id?: string;
```

Comment:
isn't that mandatory?

Reply:
Interface changes:

create_bot - remove support for webhooks, add new param owner_client_id (required when authorizing via PATs; ignored otherwise)